w09h03/test/pgdp/filter/DGTest.java

```diff
@@ -0,0 +1,5539 @@
+                "\n" +
+                "  \n" +
+                "- Supposed to be less calories.\n" +
+                "- Bye.\n" +
+                "\n" +
+                "  \n" +
+                "I gotta say something.\n" +
+                "\n" +
+                "  \n" +
+                "She saved my life.\n" +
+                "I gotta say something.\n" +
+                "\n" +
+                "  \n" +
+                "All right, here it goes.\n" +
+                "\n" +
+                "  \n" +
+                "Nah.\n" +
+                "\n" +
+                "  \n" +
+                "What would I say?\n" +
+                "\n" +
+                "  \n" +
+                "I could really get in trouble.\n" +
+                "\n" +
+                "  \n" +
+                "It's a bee law.\n" +
+                "You're not supposed to talk to a human.\n" +
+                "\n" +
+                "  \n" +
+                "I can't believe I'm doing this.\n" +
+                "\n" +
+                "  \n" +
+                "I've got to.\n" +
+                "\n" +
+                "  \n" +
+                "Oh, I can't do it. Oome on!\n" +
+                "\n" +
+                "  \n" +
+                "No. Yes. No.\n" +
+                "\n" +
+                "  \n" +
+                "Do it. I can't.\n" +
+                "\n" +
+                "  \n" +
+                "How should I start it?\n" +
+                "\"You like jazz?\" No, that's no good.\n" +
+                "\n" +
+                "  \n" +
+                "Here she comes! Speak, you fool!\n" +
+                "\n" +
+                "  \n" +
+                "Hi!\n" +
+                "\n" +
+                "  \n" +
+                "I'm sorry.\n" +
+                "\n" +
+                "  \n" +
+                "- You're talking.\n" +
+                "- Yes, I know.\n" +
+                "\n" +
+                "  \n" +
+                "You're talking!\n" +
+                "\n" +
+                "  \n" +
+                "I'm so sorry.\n" +
+                "\n" +
+                "  \n" +
+                "No, it's OK. It's fine.\n" +
+                "I know ";
+
+        String actual = builder.toString();
+
+        Assertions.assertEquals(shouldBe, actual);
+
+        fp.close();
+    }
+
+    @Test
+    @DisplayName("Test DG 03 Decode neues BufferedImage 8x8 mit Buchstabe e")
+    public void test03() throws IOException{
+        int size = 8;
+        BufferedImage image = new BufferedImage(size,size,BufferedImage.TYPE_INT_ARGB);
+        for(int i = 0; i < size; i++){
+            for(int j = 0; j < size; j++){
+                image.setRGB(i,j,Color.YELLOW.getRGB());
+            }
+        }
+        int[] colorsE = new int[]{Color.BLACK.getRGB(), Color.WHITE.getRGB(), Color.WHITE.getRGB(),
+                Color.BLACK.getRGB(), Color.BLACK.getRGB(), Color.WHITE.getRGB(), Color.BLACK.getRGB(), Color.WHITE.getRGB()};
```

Comment:
Could you extract this into a helper method, like rgbArrayFromColors(Color... colors).